source/_docs/configuration/templating.markdown

```diff
@@ -644,3 +644,13 @@ The default priority of operators is that the filter (`|`) has priority over eve
 {% endraw %}
 
 Would round `10` to 2 decimal places, then divide `states('sensor.temperature')` by `10` (rounded to 2 decimal places so 10.00). This behavior is maybe not the one expected, but priority rules imply that.
+
+To get the (possibly) desired result use parenthesis like this:
```

Comment:
This feels strange to add, it's the same as described above it

Reply:
@ludeeus are you talking about the phrasing or the example? I had some problems finding the correct syntax for this and wondered why it does not round the result. It would be good to have an example in the documentation.

Reply:
Both I guess, this is what this section already states: "filter (`|`) has priority over everything except brackets."

Reply:
yes, I can see your point. I for myself know the operator priority now. So if you think that example isn't needed, feel free to close this pull request.

Reply:
Looking at this, I kinda agree with Ludeeus: Its the same as the above. I don't think we should add this.